documentation/settings/topics/service-now-cmdb-sync.rst

```diff
@@ -0,0 +1,234 @@
+
+=============
+Configuration
+=============
+
+Enter the information prepared earlier in their corresponding configuration field.
+
+.. image:: /settings/media/servicenow_cmdb_configuration.png
+   :width: 1917
+   :alt: ServiceNow CMDB integration configuration
+
+The Last Sync Date, if populated, notes the last timestamp sync has started and finished successfully. This timestamp is used to sync changed assets since the last successful run.
+
+To force a full sync the next time the process runs, reset the last sync date.
+
+Scheduling the sync is optional and if a schedule is selected, the time the process runs will be within plus or minus 30 minutes of the selected schedule configuration.
+
+=====================
+Supported asset types
+=====================
+
+- Blade Enclosure
+- Blade Network
+- Blade Server
+- Blade Storage
+- Environmental
+- KVM Switch
+- Monitor
+- Network Device
+- Network Storage
+- Rack
+- Rack PDU
+- Server
+- Small UPS
+- Transfer Switch
+- UPS
+
+=============
+Field mapping
+=============
+
+.. list-table::
+   :header-rows: 1
+   :align: center
+   :class: datatable display compact
+
+   * - Hyperview
+     - Import Set
+     - Data Type
+     - Required
+   * - hyperview_id
+     - u_hyperview_id
+     - String
+     - Yes
+   * - name
+     - u_name
+     - String
+     - Yes
+   * - hyperview_asset_type
+     - u_hyperview_asset_type
+     - String
+     - Yes
+   * - serial_number
+     - u_serial_number
+     - String
+     - Yes
+   * - model
+     - u_model
+     - String
+     - Yes
+   * - manufacturer
+     - u_manufacturer
+     - String
+     - Yes
+   * - location_path
+     - u_location_path
+     - String
+     - Yes
+   * - room_location
+     - u_room_location
+     - String
+     - Yes
+   * - room_location_id
+     - u_room_location_id
+     - String
+     - Yes
+   * - lifecycle_state
+     - u_lifecycle_state
+     - String
+     - Yes
+   * - dns_hostname
+     - u_dns_hostname
+     - String
+     - No
+   * - rack_location
+     - u_rack_location
+     - String
+     - No
+   * - rack_location_id
+     - u_rack_location_id
+     - String
+     - No
+   * - rack_elevation
+     - u_rack_elevation
+     - Int/numeric
+     - No
+   * - rack_side
+     - u_rack_side
+     - String
+     - No
+   * - operating_system
+     - u_operating_system
+     - String
+     - No
+   * - power_providing_assets
+     - u_power_providing_assets
+     - String
+     - No
+   * - power_providing_asset_ids
+     - u_power_providing_asset_ids
+     - String
+     - No
+
+Note that when there are ID and name fields returned as an array, the sorting order will be the same. For example, if the asset has two power-providing assets, the name and ID order in their respective arrays will be the same. This is done to make it easier to write data reconciliation rules on the ServiceNow side.
+
+ImportSet JSON Example
```

Comment:
Should be a lower case "e" in "Example".

Reply:
Fixed, Thanks!